src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2025,6 +2025,10 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_CreateFalseMaskAll:
+                GetEmitter()->emitInsSve_R(ins, emitSize, targetReg, opt);
+                break;
+
```

Comment:
Both this and `CreateTrueMaskAll` feel like cases we should never encounter; as they always should have been converted to `CNS_MSK`